e2e/libs/workload/workload.py

```diff
@@ -19,6 +19,12 @@ def create_storageclass(name):
 
     with open(filepath, 'r') as f:
         manifest_dict = yaml.safe_load(f)
+        manifest_dict['metadata']['name'] = name
+        # correct replica count
+        if name == 'longhorn-test-1-replica':
```

Comment:
Is it possible to adopt this mechanism https://github.com/longhorn/longhorn-tests/pull/1867/files#diff-6422ed87ee022691c861ebd060f6651dd79d25bef034662a78dc201f40a8838bR14 to make storage class accept arbitrary option settings instead of hard-coding it?